assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -37,6 +37,10 @@
 CONTEXT = "context"
 GROUND_TRUTH = "ground_truth"
 CORRELATION_ID = "correlationid"
+TRACE_ID = "traceid"
+ROOT_SPAN = "rootspan"
+
+PASSTHROUGH_COLUMNS = [CORRELATION_ID, TRACE_ID, ROOT_SPAN]
```

Comment:
nit: this correlation_id is an old schema so should be deprecated

Reply:
I know I'm just supporting everything possible for backwards compat - I don't think this will break anything.  We can always remove it sometime later in like a year when we know it hasn't been used in forever.